File: EventsVendor.UI/src/models/Ticket.ts
export interface Ticket {
	id: number;
	eventId: number;
	userId: string;
	bookingDate: string;
	status: number;
}

export interface BookTicketResponse {
	id: number;
	eventId: number;
	userId: string;
	bookingDate: string;
	status: ticketStatus;
}

export enum ticketStatus {
	Cancelled = 0,
	Booked = 1,
}

File: EventsVendor.API/Models/Ticket.cs
using System;
using System.Text.Json.Serialization;

namespace EventsVendor.Models
{
    public class Ticket
    {
        public int Id { get; set; }

        public int EventId { get; set; }

        [JsonIgnore]
        public Event Event { get; set; }

        public string UserId { get; set; }

        [JsonIgnore]
        public EventsVendorUser User { get; set; }

        public DateTime BookingDate { get; set; }

        public TicketStatus Status { get; set; }

       
    }

    public enum TicketStatus
    {
        Booked,
        Cancelled,
    }   
}